source/Engine.cpp

```diff
@@ -1032,7 +1032,7 @@ void Engine::Step(bool isActive)
 void Engine::Go()
 {
 	++step;
-	currentCalcBuffer = currentCalcBuffer ? 0 : 1;
+	currentCalcBuffer = (!currentCalcBuffer) & 1;
```

Comment:
If you do boolean toggles, do it with booleans. Which is actually what we had before #9522.

Reply:
Wow, just looked through that.